src/error.rs

```diff
@@ -1,13 +1,20 @@
 use holochain_conductor_api::ExternalApiWireError;
 use std::error::Error;
 
-#[derive(Debug)]
+#[derive(Debug, thiserror::Error)]
```

Comment:
Fed up with this not being an error. Every usage has to `map_err` and convert to a proper error type